connect/runtime/src/main/java/org/apache/kafka/connect/runtime/Herder.java

```diff
@@ -82,7 +82,7 @@
     void connectorConfig(String connName, Callback<Map<String, String>> callback);
 
     /**
-     * Set the configuration for a connector. This supports creation, update, and deletion.
+     * Set the configuration for a connector. This supports creation and updating.
```

Comment:
Looking at this a little more, `ConnectorsResource.putConnectorConfig()` would throw NPE if the config is null, so I don't think it could have been used for deletion. So maybe we're OK?